src/Counter.elm

```diff
@@ -0,0 +1,97 @@
+module Counter exposing (main)
```

Comment:
`main` isn't very useful outside of this module.
You should export `init`, `update`, `view`, `Msg` and `Model`, allowing external modules to use whatever they want without breaking anything private.